cl/alerts/tasks.py

```diff
@@ -162,34 +165,45 @@ def send_alert_and_webhook(
     d_pk: int,
     since: datetime,
     recap_email_recipients: list[str] = None,
+    docket_entries: list[DocketEntry] = None,
 ) -> None:
     """Send an alert and webhook for a given docket
 
     :param d_pk: The docket PK that was modified
     :param since: If we run alerts, notify users about items *since* this time.
     :param recap_email_recipients: The recap.email addresses if needed to send
     the first case-user notification email.
+    :param docket_entries: A list of docket entries used if we need to send an
+    alert again for a recap.email user independently
     :return: None
     """
 
+    for_recap_email_user = False
+    if docket_entries:
+        for_recap_email_user = True
     if recap_email_recipients is None:
         recap_email_recipients = []
```

Comment:
This is to avoid using `[]` as a default arg, right? If so, it should be the *very* first thing in the function.

Reply:
Yes, that's correct. I've moved it to the top.